lib/index.js

```diff
@@ -0,0 +1,261 @@
+'use strict';
+
+const path = require('path');
+const querystring = require('querystring');
+const fetch = require('node-fetch');
```

Comment:
That is exactly how it was done before - with a model extending hof-model, it just felt like overkill when all we want to do is `fetch`